draft-ietf-tls-tls13.md

```diff
@@ -2116,6 +2128,9 @@ hash
   SHA-224, SHA-256, SHA-384, and SHA-512 {{SHS}}, respectively.  The
   "none" value is provided for future extensibility, in case of a
   signature algorithm which does not require hashing before signing.
+  The usage of MD5 and SHA-224 are deprecated. The md5_RESERVED and
+  sha224_RESERVED values MUST NOT be offered or negotiated by any
+  implementation.
```

Comment:
I think this should be stronger. TLS 1.3 agents MUST reject if any TLS 1.3-advertising agent offers them. wdyt?